### ISSUE TITLE
do not works with array of strings

### ISSUE THREAD

**Issue by vedmalex** [APIs-guru/graphql-lodash#7]
the query 
```graphql
{
  Films: allFilms(first:1) @_(get:"films"){
    films {
      title
      producers @_(join:",")
    }
  }
}
```
returns 
```json
{
  "data": {
    "Films": [
      {
        "title": "A New Hope",
        "producers": "[object Object],[object Object]"
      }
    ]
  }
}
```
instead of
```json
{
  "data": {
    "Films": [
      {
        "title": "A New Hope",
        "producers": "Gary Kurtz,Rick McCallum"
      }
    ]
  }
}
```

**Comment by vedmalex** [comment]
@IvanGoncharov? will you update the application on the server?

**Comment by IvanGoncharov** [comment]
@vedmalex Done + I released changes as `1.3.2` on NPM.
